op-acceptance-tests/tests/interop/interop_test_helpers.go

```diff
@@ -126,3 +129,82 @@ func SetupDefaultInteropSystemTest(l2ChainNums int) ([]validators.WalletGetter,
 	}
 	return walletGetters, totalValidators
 }
+
+// FundWalletsFromFaucet funds wallet and returns txplan options which encapsulates funded wallet
+func FundWalletFromFaucet(ctx context.Context, logger log.Logger, sys system.InteropSystem, chainIdx int, faucetOpt txplan.Option, amount *big.Int) (txplan.Option, error) {
+	if len(sys.L2s()) < chainIdx {
+		return nil, fmt.Errorf("invalid chain idx: %d", chainIdx)
+	}
+	privateKey, err := crypto.GenerateKey()
+	if err != nil {
+		return nil, err
+	}
+	wallet, err := system.NewWalletV2(ctx, sys.L2s()[chainIdx].Nodes()[0].RPCURL(), privateKey, nil, logger)
+	if err != nil {
+		return nil, err
+	}
+	to := crypto.PubkeyToAddress(privateKey.PublicKey)
+	opt := DefaultTxOpts(wallet)
+	tx := txplan.NewPlannedTx(faucetOpt, txplan.WithValue(amount), txplan.WithTo(&to))
+	_, err = tx.Included.Eval(ctx)
+	if err != nil {
+		return nil, err
+	}
+	logger.Info("funded", "address", to.Hex(), "chainID", sys.L2s()[chainIdx].ID())
```

Comment:
```suggestion
	logger.Info("Funded", "address", to, "chainID", sys.L2s()[chainIdx].ID())
```
Nit again, simple logging attributes are fine.
And maybe `chain := sys.L2s()[chainIdx]`, at the top of the function, to reuse, makes sense.

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/15049/commits/486c6039d09276b603613142a84197b75e05188a